src/store/toast-store.js

```diff
@@ -57,4 +57,9 @@ store.registerHandler('CHECKOUT_SUCCESS', () => {
     store.emitChange();
 });
 
+store.registerHandler('ITEM_DELETED', data => {
+    addToast(`Item ${data.itemAddress} was deleted successfully!`);
```

Comment:
Yes, but I want to know what deleted, just like I want to know what item I created (as what was done in the liked PR/comment).